Src/Clients/WebUI/Controllers/Storage/ActorsWebUiController.cs

```diff
@@ -85,5 +96,55 @@ public async Task<IActionResult> DeleteActor(int id)
 
             return Ok();
         }
+        [HttpGet]
+        public async Task<IActionResult> More(int id)
+        {
+            var model = await _authorizeApiTools.FetchAsync<ActorBindingModel>($"{CommonClientsDefaults.WebApiActorsControllerGet}/{id}", JwtToken);
+            
+            var tmp = await _authorizeApiTools.FetchAsync<List<ActorPhotoBindingModel>>($"{CommonClientsDefaults.WebApiApiActorsPhotoControllerGetAllByActor}/{id}", JwtToken);
+            model.Photos = tmp;
+
+            tmp.ForEach(async e =>
+            {
+                var fetched = await (await new HttpClient().GetAsync($"{CommonClientsDefaults.WebApiApiActorsPhotoControllerGet}/{e.PhotoId}").ConfigureAwait(false)).Content.ReadAsByteArrayAsync().ConfigureAwait(false);
+                string basePath = Path.Combine(Directory.GetCurrentDirectory(), "actorsPhotosDir");
+                var path = $@"{basePath}\{e.Path}";
+                await System.IO.File.WriteAllBytesAsync(path, fetched).ConfigureAwait(false);
+            });
+
+            await Task.Delay(20);
+            return View(model);
+        }
+        [HttpGet]
+        public IActionResult AddPhoto(int id)
+        {
+            var model = id;
+            return View(model);
+        }
+        [HttpPost]
+        public async Task<IActionResult> UploadPhoto(IEnumerable<IFormFile> photosUploadMulti, [FromForm] int id)
+        {          
+            
+            foreach (var item in photosUploadMulti)
+            {
+                if (item == null) continue;
+                byte[] bytes = new byte[item.OpenReadStream().Length + 1];
+                item.OpenReadStream().Read(bytes, 0, bytes.Length);
+                var content = new ByteArrayContent(bytes);
+                content.Headers.Add("name", "photosUploadMulti");
+                MultipartFormDataContent formDataContent = new MultipartFormDataContent();
+                formDataContent.Add(content, "file", item.FileName);
+                var client = new HttpClient();
+                var result = client.PostAsync($"{CommonClientsDefaults.WebApiApiActorsPhotoControllerPhotoUpload}/{id}", formDataContent).Result;
+                //return StatusCode((int)result.StatusCode);
+                //await _authorizeApiTools.PostAsync<ActorBindingModel>($"{CommonClientsDefaults.WebApiApiActorsPhotoControllerPhotoUpload}/{id}",
+                 //formDataContent, JwtToken);
+            }
+
+            return RedirectToAction("Index");            //return RedirectToAction("More",id);//($"{ CommonClientsDefaults.WebApiActorsControllerGet}/{id}");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/liannoi/team-project/pullRequest?prid=5384135)